cmd/fyne/internal/commands/release.go

```diff
@@ -149,6 +148,13 @@ type Releaser struct {
 	password     string
 }
 
+// NewReleaser returns a command that can handle the packaging a GUI apps for release from local Fyne source code.
+func NewReleaser() *Releaser {
+	r := &Releaser{}
+	r.appData = &appData{}
+	return r
```

Comment:
Should we not do it like this to be consistent with the others?
```suggestion
	return &Releaser{appData: &appData{}}
```

Reply:
No, that won't work - the appData is not a field of Releaser but inherited from a parent.

Reply:
Ah, I see. Alright then. That is honestly one of the few things with Go that I strongly dislike